go/vt/vttablet/tabletserver/txthrottler/tx_throttler.go

```diff
@@ -101,6 +106,19 @@ type TopologyWatcherInterface interface {
 // go/vt/throttler.GlobalManager.
 const TxThrottlerName = "TransactionThrottler"
 
+// fetchKnownCells gathers a list of known cells from the topology. An ErrFoundNoCells
+// error is returned if the GetKnownCells call to topoServer returns no cells.
+func fetchKnownCells(topoServer *topo.Server) ([]string, error) {
```

Comment:
please add a `context.Context` param, and pass the context from `healthChecksProcessorFactory`